// angular-date-format/bundles/angular-date-format.umd.js
(function (global, factory) {
	typeof exports === 'object' && typeof module !== 'undefined' ? factory() :
	typeof define === 'function' && define.amd ? define(factory) :
	(factory());
}(this, (function () { 'use strict';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes} checked by tsc
 */
/**
 * Generated bundle index. Do not edit.
 */
Object.defineProperty(exports, "__esModule", { value: true });
var public_api_1 = require("./public_api");
exports.DateFormatComponent = public_api_1.DateFormatComponent;
exports.MatDateFormatComponent = public_api_1.MatDateFormatComponent;
exports.DateFormatModelMock = public_api_1.DateFormatModelMock;
exports.DateFormatValidatorMock = public_api_1.DateFormatValidatorMock;
exports.DateFormatEnum = public_api_1.DateFormatEnum;
exports.DateFormatModel = public_api_1.DateFormatModel;
exports.DateFormatValidator = public_api_1.DateFormatValidator;
exports.DateFormatModule = public_api_1.DateFormatModule;
exports.MatDateFormatModule = public_api_1.MatDateFormatModule;
var base_date_format_component_1 = require("./src/reusable/base-date-format.component");
exports.ɵa = base_date_format_component_1.BaseDateFormatComponent;
var base_field_component_1 = require("./src/reusable/base-field.component");
exports.ɵb = base_field_component_1.BaseFieldComponet;

})));
